src/app.js

```diff
@@ -68,11 +99,11 @@ io.on('connection', function(socket){
   socket.on('live', function(formData) {
     var req = {body: formData};
     generator.createDistDir(req, socket, function(appFolder, url) {
-      socket.emit('live.ready', {
-        appDir: appFolder,
-        url: url
-      });
-      //generator.showLivePreview(req, socket);
+        folder = appFolder;
```

Comment:
@aayusharora Actually, the folder variable was used to store the name of the folder whose contents were going to be uploaded and committed to Github. But it is not that good approach, so I did it other way.